### ISSUE TITLE
Configure user for monitoring database

### ISSUE THREAD

**Issue by JohnDBallentineIII** [informix/informixhq-server#14]
I'm able to connect to the live data as a named user, but I can't configure a user to connect to the monitoring database - it's trying to use my Windows credentials, but I use a Linux system user.

I don't see any option to configure this in the GUI, or in the documented options for the server.properties file

**Comment by ekuehnhausen** [comment]
When you started the agent with a repository.url, what user is being used? Did you omit the USER and PASSWORD from the URL and rely OS authentication with the user that started the agent?

**Comment by JohnDBallentineIII** [comment]
That is correct.  Does it need to be specified there to be picked up by the server part?

**Comment by ekuehnhausen** [comment]
Yes, at this time, the server gets the repository url from the agent. This is something that will need to be addressed in a future release. But at this time, I think the only way to make it work is to have the user/password in the repository url.
